config/environments/development.rb

```diff
@@ -67,12 +67,13 @@
   config.api.v1                                  = ActiveSupport::OrderedOptions.new
   config.api.v1.connection_options               = ActiveSupport::OrderedOptions.new
   config.api.v1.connection_options.namespace     = 'Limber'
-  config.api.v1.connection_options.url           = ENV.fetch('API_URL', "http://#{localhost}:3000/api/1/")
+  config.api.v1.connection_options.url           = ENV.fetch('API_URL', "http://#{localhost_env}:3000/api/1/")
   config.api.v1.connection_options.authorisation = ENV.fetch('API_KEY', 'development')
 
   config.api.v2                                  = ActiveSupport::OrderedOptions.new
   config.api.v2.connection_options               = ActiveSupport::OrderedOptions.new
-  config.api.v2.connection_options.url           = ENV.fetch('API2_URL', "http://#{localhost}:3000/api/v2")
+  config.api.v2.connection_options.url           = ENV.fetch('API2_URL', "http://#{localhost_env}:3000/api/v2")
+  config.api.v2.connection_options.js_url        = ENV.fetch('API2_URL', 'http://localhost:3000/api/v2')
```

Comment:
There something with these using the same API2_URL env that I dont see very clear. Is it a mistake or is it because of any particular reason?

Reply:
No mistake.  If the APIv2 URL were for a different server -- a proper domain name.  You wouldn't want server side requests going to a different instance of SequenceScape than the browser requests.  This problem only exists when you need to talk to the docker host from the server and to carry on using `localhost` in the browser.

Reply:
What wouldn't work, admittedly, is if you specified `http://localhost:1234/api/v2` in the `API2_URL` environment variable.  Maybe the substitution needs to be made to the fetched variable, whether it's default or specified.  I'll update for that.